speech_recognition/__init__.py

```diff
@@ -713,6 +724,13 @@ def recognize_google_cloud(self, audio_data, credentials_json=None, language="en
         assert isinstance(language, str), "``language`` must be a string"
         assert preferred_phrases is None or all(isinstance(preferred_phrases, (type(""), type(u""))) for preferred_phrases in preferred_phrases), "``preferred_phrases`` must be a list of strings"
 
+        # Implementation of assertions of common api_params
```

Comment:
Could you remove assertions?

ref: https://realpython.com/python-assert-statement/#when-not-to-use-assertions
>In general, you shouldn’t use assertions for data processing or data validation

This repository has repeatedly made the mistake of using assertions for data validation, and I don't want to add new ones as a maintainer

Reply:
That's fair, i removed the antipattern. I hope the code is now acceptable :)